package.json

```diff
@@ -22,6 +22,8 @@
     "lint:style": "stylelint --fix '**/*.scss' && echo 'Style linting complete'",
     "prepublishOnly": "npm run type-check & npm run lint && npm run test && npm run build",
     "storybook": "start-storybook -c ./config/storybook -p 6006",
+    "build-storybook": "build-storybook -c ./config/storybook",
```

Comment:
Very minor: could we amend to "storybook:build" and "storybook:deploy" to follow the convention?